src/Runner.Worker/ExecutionContext.cs

```diff
@@ -106,6 +106,7 @@ public sealed class ExecutionContext : RunnerService, IExecutionContext
     {
         private const int _maxIssueCount = 10;
         private const int _throttlingDelayReportThreshold = 10 * 1000; // Don't report throttling with less than 10 seconds delay
+        private const string _noticeLogPrefix = "\u001b[31m"; //white text
```

Comment:
we should be consistent with error/warning. iirc today that means the log view (recognizes lines that start with `[error]` or `[warning]`)

also curious if you set the ANSI color code, do you need to reset it at the end of the line? You might need to download the log and open in a text viewer to see.

Reply:
If we switch to ANSI color codes we should be consistent and update error/warning too.

Reply:
+1 make it consistent with error/warning.